src/lib/api_retry.ts

```diff
@@ -0,0 +1,36 @@
+import { AxiosInstance } from "@mittwald/api-client-commons";
+import debug from "debug";
+import axiosRetry from "axios-retry";
+
+const d = debug("mw:api-retry");
+
+export function configureAxiosRetry(axios: AxiosInstance) {
+  axios.interceptors.request.use((config) => {
+    return {
+      ...config,
+      validateStatus: (status) => status < 300,
+    };
+  });
+
+  axiosRetry(axios, {
+    retries: 50,
```

Comment:
Mir erscheint `50` etwas viel.

Reply:
```suggestion
    retries: 10,
```

Ich setze es mal auf 10 runter-- wir brauchen auf jeden Fall mehr als die standardmäßigen 3 Retries, da manche Routen tatsächlich _echt_ lange brauchen, bis sich die ACLs aktualisiert haben (unabhängig davon, dass wir das Issue intern natürlich sowieso fixen sollten).